.github/workflows/fips-checksums.yml

```diff
@@ -73,3 +73,29 @@ jobs:
         with:
           name: fips_checksum
           path: artifact/
+  verify-checksums:
```

Comment:
It's not clear this test is doing much for us anymore, and isn't redundant with above

Reply:
It is pointless for us.